src/components/translatable/RowHeader.js

```diff
@@ -112,7 +113,7 @@ export default function RowHeader({
   const defaultHeader = useDeepCompareMemo(() => (
     <div style={styles.defaultHeader}>
       <Typography variant='h6' style={styles.title}>
-        {`${bookId} ${chapter}:${verse}`}
+        {`${bookId} ${chapter}:${verse}`.toUpperCase()}
```

Comment:
![image](https://user-images.githubusercontent.com/118073/161720395-b1bf5585-1daf-4ad0-9c72-dbfa9c338447.jpeg)

In the above screenshot, it looks like the whole string is uppercased. I believe just the bookId needed. That way the front:intro isn't the same as the GEN.